homeassistant/components/binary_sensor/__init__.py

```diff
@@ -35,6 +35,7 @@
     'smoke',         # Smoke detector
     'sound',         # On means sound detected, Off means no sound
     'vibration',     # On means vibration detected, Off means no vibration
+    'plug',          # On means plugged in, Off means plugged off
```

Comment:
I think "unplugged" is the correct term for off in this case.

Reply:
Thanks, I've updated this here and in the  documentation as well.